src/CMakeLists.txt

```diff
@@ -46,6 +40,26 @@ foreach(domain ${TARGET_DOMAINS})
 endforeach()
 
 # Generate header with enabled backends for testing
+
+function(generate_header_file)
```

Comment:
The lines inside the functions should be indented with 2 spaces, like the `if()`

Reply:
fixed in [9ae43f0](https://github.com/oneapi-src/oneMKL/pull/571/commits/9ae43f0036d3bef1060383a467720988576fa5d6)